### PR TITLE
`Development`: Document exam exercise submission policies

### PR DESCRIPTION
**Pull request by terlan98**
#### General
- [x] This is a small issue that I tested locally and was confirmed by another developer on a test server.
- [x] Language: I followed the [guidelines for inclusive, diversity-sensitive, and appreciative language](https://ls1intum.github.io/Artemis/dev/guidelines/language-guidelines/).
- [x] I chose a title conforming to the [naming conventions for pull requests](https://ls1intum.github.io/Artemis/dev/development-process/#naming-conventions-for-github-pull-requests).

### Motivation and Context
Document changes implemented in https://github.com/ls1intum/Artemis/pull/7128

### Description
Added documentation for students and instructors.

### Steps for Testing

1. Read the exam mode docs: https://artemis-platform--7340.org.readthedocs.build/en/7340/user/exam_mode.html
2. Read the exam mode features page: {test-server-url}/features/instructors

#### Code Review
- [x] Code Review 1
- [x] Code Review 2

### CODE REVIEW

**Review by Strohgelaender — APPROVED** [review]

**Review by b-fein — APPROVED** [review]

**Review by krusche — CHANGES_REQUESTED** [review]
Could you please also mention this feature on the website https://artemis.in.tum.de/features/instructors ?

File: src/main/webapp/i18n/de/featureOverview.json [hunk 1]
```diff
@@ -85,6 +85,11 @@
...
+                },
+                "submissionPolicy": {
+                    "title": "Abgaberichtlinie Definieren",
+                    "shortDescription": "Entmutige Versuch-und-Irrtum-Ansätze mit Einreichungsstrafen und Repository-Sperrmaßnahmen.",
+                    "descriptionTextOne": "Abhängig von den Aufgaben und deinem Unterrichtsstil kann es sinnvoll sein, eine Abgaberichtlinie zu definieren. Im Allgemeinen bekämpfen die Richtlinien zur Sperrung des Speichers und zur Bestrafung von Einsendungen Lösungsansätze, die auf Versuch und Irrtum beruhen. Wenn der Schüler das Einsende-Limit erreicht, verhindert die erste Methode weitere Einsendungen, während die zweite Methode den Schüler bestraft. Die Abgaberichtlinien werden zunächst bei der Erstellung einer Programmierübung festgelegt und können später in der Benotungskonfiguration der jeweiligen Programmierübung angepasst werden."
```
Comment (b-fein) [inline src/main/webapp/i18n/de/featureOverview.json:92]:
```suggestion
                    "descriptionTextOne": "Abhängig von den Aufgaben und deinem Unterrichtsstil kann es sinnvoll sein, eine Abgaberichtlinie zu definieren. Im Allgemeinen bekämpfen die Richtlinien zur Sperrung des Repositories und zur Bestrafung von Einsendungen Lösungsansätze, die auf Versuch und Irrtum beruhen. Wenn Studierende das Einsende-Limit erreichen, verhindert die erste Methode weitere Einsendungen, während die zweite Methode einen Punktabzug zur Folge hat. Die Abgaberichtlinien werden zunächst bei der Erstellung einer Programmierübung festgelegt und können später in der Benotungskonfiguration der jeweiligen Programmierübung angepasst werden."
```
- I’d not translate Repository. That should be well-enough known.
- Schüler → gender neutral ‘Studierende’
- (optional, might make sense in the English translation, too) Penalize: more concrete and call it a points deduction

**Review by b-fein — COMMENTED** [review]
Looks good, I’ve got a few suggestions for the German translation, though.

File: src/main/webapp/i18n/de/featureOverview.json (hunk 1 above)
Comment (terlan98) [reply src/main/webapp/i18n/de/featureOverview.json:92]: Thanks!

**Review by b-fein — APPROVED** [review]

**Review by b-fein — APPROVED** [review]

**Review by LenaKahle — APPROVED** [review]
Looks good.

**Pull request merged**